lib/node_modules/@stdlib/string/base/replace-after/README.md

```diff
@@ -0,0 +1,139 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2023 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# replaceAfter
+
+> Replace the substring after the first occurrence of a specified search string.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var replaceAfter = require( '@stdlib/string/base/replace-after' );
+```
+
+#### replaceAfter( str, replacement, search\[, fromIndex] )
```

Comment:
@HarshitaKalani I just noticed this, but the argument order is reversed from [`@stdlib/string/replace-before`](https://github.com/stdlib-js/stdlib/blob/a7d8f429e9058cebfb4931309d5712b05f8c7931/lib/node_modules/%40stdlib/string/replace-before/README.md). Is there a reason for this? Personally, I would expect the argument order to be the same for the sake of API consistency.

Reply:
I kept it that way since there's an extra argument 'fromIndex', so thought it's better to keep search at last in case fromIndex is also given as the argument. 
Should I keep it as replaceAfter(str, search, replacement\[, fromIndex] ) or replaceAfter(str, search\[, fromIndex], replacement) would be better?

Reply:
Given that [`@stdlib/string/replace`](https://github.com/stdlib-js/stdlib/tree/9fac15ccfdf4b3b1b5b92442e56b077c6b69fa42/lib/node_modules/%40stdlib/string/replace#replace-str-search-newval-) is

```text
replace( str, search, newval )
```

and [`@stdlib/string/replace-before`](https://github.com/stdlib-js/stdlib/tree/9fac15ccfdf4b3b1b5b92442e56b077c6b69fa42/lib/node_modules/%40stdlib/string/replace-before#replacebefore-str-search-replacement-) is

```text
replaceBefore( str, search, replacement )
```

My preference is to keep consistency where `fromIndex` is the last argument and `search` comes before `replacement`.

```text
replaceAfter( str, search, replacement[, fromIndex] )
```

The `fromIndex` argument comes last in other APIs (e.g., [`@stdlib/string/starts-with`](https://github.com/stdlib-js/stdlib/tree/9fac15ccfdf4b3b1b5b92442e56b077c6b69fa42/lib/node_modules/%40stdlib/string/starts-with) and others), so IMO best to keep consistent here and have it be the last argument, which is also better from a polymorphic interface standpoint. An optional argument in the middle of mandatory arguments is generally harder to optimize.